build.gradle

```diff
@@ -128,6 +128,18 @@ task discoJar(type: com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar)
   exclude 'META-INF/*.RSA'
 }
 
+task gapicJar(type: com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar) {
```

Comment:
Maybe name the task more generally - `fatJar`. 

I have no opinion on with/without main class attribute (yet).